_posts/2020-02-10-Baby-Steps.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: blog
+author: Will Holmgren
+---
+
+Are you curious about the Solar Forecast Arbiter but you don’t want to
+commit or you’re not sure how to get started? This is for you!
+
+First, we totally understand your hesitation. Full use of the Solar
+Forecast Arbiter requires a lot of trust in the technical aspects of the
+platform and in the platform administrators. We expect to build that
+trust over time. The Solar Forecast Arbiter also has a bit of a learning
+curve, and we expect users to build expertise over time.
+
+So, we’d like to propose a set of baby steps that stakeholders can take
+to build trust and expertise.
+
+**Step 0**: Make a free, no-obligation, no-lawyers-required user account at
+[dashboard.solarforecastarbiter.org](https://dashboard.solarforecastarbiter.org). Then:
+  * Explore the reference data, reference forecasts, and the daily updated
+reports for select reference data and forecasts.
+  * Download some reference data, forecasts, or report metrics.
+
+*Do this today!*
+
+If you like what you see, consider approaching the Arbiter in the following steps:
+
+1. Have an authorized representative at your organization sign the no-obligation,
+   [Data Use Agreement](https://solarforecastarbiter.org/assets/45864%20Approved_Final%20version%201.1.pdf):
+      * You cannot utilize the full functionality of the platform until this
+        step has been completed
+      * Signing the DUA is free and does NOT obligate you to share any data.
+        It is not a data sharing agreement, rather it is an agreement about
+        how the framework, and data on it, may be used.
+      * You are **never** obligated to upload any data to the platform.
+        When you do upload data, the default is that it is private.
+        Only data owners can change sharing privileges.  As a data owner
+        you are in control of who sees your data and you can
+        remove it at any time.
+2. Make some dummy metadata, upload some dummy data, make an analysis report
+   for reference sites and/or dummy data.
+3. Share private dummy data with one or two other users within and outside
+   your organization to get an understanding of how data sharing and privacy
+   works.
+4. Add real observational or forecast metadata, and upload real data for one
+   or two sites.
+5. Use the Arbiter to produce an analysis report for a small, real world
+   forecast problem.
```

Comment:
Should these points have links to the documentation on how to complete these steps?